README.md

```diff
@@ -7,25 +7,41 @@ Docker-Locust
 
 The purpose of this project is to provide a ready and easy-to-use version of [locust.io] which also contains additional/useful features that are required.
 
+Architecture
+------------
+Docker-Locust consist of 3 different roles:
+
+- Master: Instance that will run Locust's web interface where you start and stop the load test and see live statistics.
+- Slave: Instance that will simulate users and attack the target url based on user parameters.
+- Controller: Instance that will be run for automatic mode and will download the HTML report at the end of load test.
+
+This architecture support following type of deployment:
+
+- single container (standalone mode): If user have only one single machine.
+- multiple containers (normal mdoe): If user have more than one machine and want to create bigger load. This type of deployment might be used in docker-swarm or kubernetes case. An example for deployment in different containers can be seen in [docker-compose].
+
 Key advantages
 --------------
 
-1. It allows users to upload load test scenario/script from different resources (any HTTP/HTTPS URL, S3 bucket, and local machine).
-2. It has the ability to be run in any CI tool e.g. Jenkins (It can start/stop load test automatically) and provides an HTML report at the end of a load test.
+1. It allows locust to read load test scenario/script from different resources (any HTTP/HTTPS URL, S3 bucket, and local machine).
+2. It has the ability to be run in any CI tool e.g. [Jenkins] (It can start/stop load test automatically) and provides an HTML report at the end of a load test.
 3. It is also possible to be deployed in AWS to create bigger load.
 
 Requirements
 ------------
 1. [docker engine] version 1.9.1+
-2. [docker-compose] version 1.6.0+
+2. [docker-compose] version 1.6.0+ (optional)
 
 Getting Started
 ---------------
 
-Run the application with the command:
+### Single machine / Standalone mode
+---
+
+docker-locust can be run as **standalone** version by passing parameter **STANDALONE=true** at the beginning of the script. Standalone version is for users who has only 1 single machine.
```

Comment:
Why don't you make standalone mode the default one? It looks like the default way we suggest to run it is still using the local.sh with positional args + docker compose.

Reply:
great idea!